src/main/java/mods/eln/gridnode/GridElement.java

```diff
@@ -64,30 +64,30 @@ private boolean onTryGridConnect(EntityPlayer entityPlayer, ItemStack stack, Ele
         }
         // Check if it's the *correct* cable descriptor.
         if (!cable.equals(desc.cableDescriptor)) {
-            Utils.sendMessage(entityPlayer, "Wrong cable, you need " + desc.cableDescriptor.name);
+            Utils.addChatMessage(entityPlayer, "Wrong cable, you need " + desc.cableDescriptor.name);
```

Comment:
I personally like "sendMessage" better as a function name. Not sure why you changed it.